.changeset/thirty-goats-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Add allowlist option to noNamespaceImport rule
```

Comment:
This isn't a patch, but a feature. The contribution guide explains what to do in case of features. The template contains a comment with a link to the relative section .

Second, the changeset is for end users, so we need to provide a better wording for this feature, some links, good markdown formatting, and an example of why and how to use it.